src/b2b/scim_connections.rs

```diff
@@ -0,0 +1,332 @@
+    /// status_code: The HTTP status code of the response. Stytch follows standard HTTP response status code
+    /// patterns, e.g. 2XX values equate to success, 3XX values are redirects, 4XX are client errors, and 5XX
+    /// are server errors.
+    #[serde(with = "http_serde::status_code")]
+    pub status_code: http::StatusCode,
+    /// connection: The `SAML Connection` object affected by this API call. See the
+    /// [SAML Connection Object](https://stytch.com/docs/b2b/api/saml-connection-object) for complete response
+    /// field details.
+    pub connection: std::option::Option<SCIMConnection>,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, Default)]
+pub enum CreateRequestIdp {
+    #[serde(rename = "unknown")]
+    #[default]
+    Unknown,
+    #[serde(rename = "okta")]
+    Okta,
+    #[serde(rename = "microsoftentra")]
+    Microsoftentra,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, Default)]
+pub enum UpdateRequestIdp {
+    #[serde(rename = "unknown")]
+    #[default]
+    Unknown,
+    #[serde(rename = "okta")]
+    Okta,
+    #[serde(rename = "microsoftentra")]
+    Microsoftentra,
+}
+
+pub struct Connections {
+    http_client: crate::client::Client,
+}
+
+impl Connections {
+    pub fn new(http_client: crate::client::Client) -> Self {
+        Self {
+            http_client: http_client.clone(),
+        }
+    }
+
+    pub async fn update(&self, body: UpdateRequest) -> crate::Result<UpdateResponse> {
+        let organization_id = &body.organization_id;
+        let connection_id = &body.connection_id;
+        let path = format!("/v1/b2b/scim/{organization_id}/connections/{connection_id}");
+        self.http_client
+            .send(crate::Request {
+                method: http::Method::PUT,
+                path,
+                body,
+            })
+            .await
+    }
+    pub async fn delete(&self, body: DeleteRequest) -> crate::Result<DeleteResponse> {
+        let organization_id = &body.organization_id;
+        let connection_id = &body.connection_id;
+        let path = format!("/v1/b2b/scim/{organization_id}/connections/{connection_id}");
+        self.http_client
+            .send(crate::Request {
+                method: http::Method::DELETE,
+                path,
+                body,
+            })
+            .await
+    }
+    pub async fn rotate_start(
+        &self,
+        body: RotateStartRequest,
+    ) -> crate::Result<RotateStartResponse> {
+        let organization_id = &body.organization_id;
+        let connection_id = &body.connection_id;
+        let path =
+            format!("/v1/b2b/scim/{organization_id}/connections/{connection_id}/rotate/start");
+        self.http_client
+            .send(crate::Request {
+                method: http::Method::POST,
+                path,
+                body,
+            })
+            .await
+    }
+    pub async fn complete(&self, body: CompleteRequest) -> crate::Result<CompleteResponse> {
```

Comment:
hmm why is this one not named `rotate_complete`? we should look into that, it will be a breaking change later